package.json

```diff
@@ -15,19 +15,19 @@
     "type-check": "nx run-many --target=type-check"
   },
   "devDependencies": {
-    "@typescript-eslint/eslint-plugin": "^5.61.0",
-    "@typescript-eslint/parser": "^5.61.0",
-    "eslint": "^8.44.0",
-    "eslint-plugin-jest": "^27.2.2",
-    "eslint-plugin-jest-dom": "^5.0.1",
-    "eslint-plugin-react": "^7.32.2",
+    "@typescript-eslint/eslint-plugin": "^5.62.0",
+    "@typescript-eslint/parser": "^5.62.0",
+    "eslint": "^8.57.0",
+    "eslint-plugin-jest": "^27.9.0",
+    "eslint-plugin-jest-dom": "^5.1.0",
+    "eslint-plugin-react": "^7.34.0",
     "eslint-plugin-react-hooks": "^4.6.0",
-    "eslint-plugin-storybook": "^0.6.12",
-    "eslint-plugin-testing-library": "^5.11.0",
+    "eslint-plugin-storybook": "^0.6.15",
+    "eslint-plugin-testing-library": "^5.11.1",
     "husky": "^8.0.3",
-    "lint-staged": "^13.2.3",
-    "nx": "^17.0.2",
-    "rimraf": "^5.0.1",
+    "lint-staged": "^13.3.0",
+    "nx": "^18.0.8",
```

Comment:
I bumped nx from v17 to v18 to transitively bump axios and follow-redirects. There were no breaking changes in nx v18